pkg/common/moerr/error.go

```diff
@@ -26,30 +27,85 @@ import (
 const MySQLDefaultSqlState = "HY000"
 
 const (
-	// 0 is OK.
-	SUCCESS uint16 = 0
-	INFO    uint16 = 1
-	WARN    uint16 = 2
+	// 0 - 99 is OK.  They do not contain info, and are special handled
+	// using a static instance, no alloc.
+	Ok              uint16 = 0
+	OkStopCurrRecur uint16 = 1
```

Comment:
should we avoid errcode in 1~256 ?
in linux [/usr/include/errno.h](https://man7.org/linux/man-pages/man3/errno.3.html), there is already define at least 133`（1~133）` errcode